# Problema 1

Considere la sucesión $a_{n} = 3n^{2} + 7$. Por medio de un código en *Python* despliega en pantalla:

1. Los primeros $5$ términos.
2. Los $5$ términos que vienen inmediatamente después del decimoquinto término.

In [4]:
a=[]
for i in range(20):
    n=i+1
    a.append(3*n**2+7)
    print(f"a_{n}={a[i]}")

a_1=10
a_2=19
a_3=34
a_4=55
a_5=82
a_6=115
a_7=154
a_8=199
a_9=250
a_10=307
a_11=370
a_12=439
a_13=514
a_14=595
a_15=682
a_16=775
a_17=874
a_18=979
a_19=1090
a_20=1207



1. Los primeros $5$ términos.

In [6]:
for i in range(5):
    n=i+1
    print(f"a_{n}={a[i]}")

a_1=10
a_2=19
a_3=34
a_4=55
a_5=82


2. Los $5$ términos que vienen inmediatamente después del decimoquinto término.

In [14]:
for i in range(14,21):
    n=i+1
    a.append(3*n**2+7)
    print(f"a_{n}={a[i]}")

a_15=682
a_16=775
a_17=874
a_18=979
a_19=1090
a_20=1207
a_21=682


# Problema 2

Considere la sucesión $g_{n} = 5n^{3}$. Implemente un código en *Python* que muestre en pantalla:

1. Los primeros $4$ términos de la sucesión.
2. Los $4$ términos que vienen inmediatamente después del octavo término.
3. Determinar si el término $40.000$ pertenece a la sucesión. Si es así, indicar en qué posición se encuentra.

1. Los primeros $4$ términos de la sucesión.

In [59]:
g=[]
for i in range(20):
    h=i+1
    g.append(5*h**3)
    print(f"a_{h}={g[i]}")

for i in range(4):
    h=i+1
    g.append(5*h**3)
    print(f"a_{h}={g[i]}")

    

a_1=5
a_2=40
a_3=135
a_4=320
a_5=625
a_6=1080
a_7=1715
a_8=2560
a_9=3645
a_10=5000
a_11=6655
a_12=8640
a_13=10985
a_14=13720
a_15=16875
a_16=20480
a_17=24565
a_18=29160
a_19=34295
a_20=40000
a_1=5
a_2=40
a_3=135
a_4=320


2. Los $4$ términos que vienen inmediatamente después del octavo término.


In [63]:
for i in range(7,14):
    h=i+1
    print(f"a_{h}={g[i]}")

a_8=2560
a_9=3645
a_10=5000
a_11=6655
a_12=8640
a_13=10985
a_14=13720


3. Determinar si el término $40.000$ pertenece a la sucesión. Si es así, indicar en qué posición se encuentra.

In [ ]:
for i in range(20):
    h=i+1
    print(f"a_{h}={g[i]}")

a_1=5
a_2=40
a_3=135
a_4=320
a_5=625
a_6=1080
a_7=1715
a_8=2560
a_9=3645
a_10=5000
a_11=6655
a_12=8640
a_13=10985
a_14=13720
a_15=16875
a_16=20480
a_17=24565
a_18=29160
a_19=34295
a_20=40000


SI pertenece el 40.000 es de a_20

# Problema 3

La sucesión de Fibonacci es una serie de números en la que cada número, desde el tercero en adelante, es la suma de los dos anteriores. Comienza con el $0$ y $1$, tal como se observa a continuación:

$$0, 1, 1, 2, 3, 5, 8, 13, ...$$

Formalmente, la sucesión se define de la siguiente manera:

\begin{split}
F(0) &= 0\\
F(1) &= 1\\
F(n) &= F(n - 1) + F(n - 2), \ \text{ para $n \geq 2$}
\end{split}

Esta sucesión se utiliza en una variedad de contextos en informática y algoritmos debido a sus propiedades matemáticas y patrones.

1. Implementa un código en *Python* que muestre en pantalla los primeros $20$ términos de la sucesión.
2. Calcula, utilizando un ciclo *for*, la suma de los primeros $30$ términos de la sucesión de Fibonacci.

1. Implementa un código en *Python* que muestre en pantalla los primeros $20$ términos de la sucesión.


2. Calcula, utilizando un ciclo *for*, la suma de los primeros $30$ términos de la sucesión de Fibonacci.

# Problema 4

Implementa un código que permita guardar en una lista la sucesión de los primeros $n$ números pares, con $n \geq 1$. Luego, determina:

1. ¿Cuál es el valor del décimo término?
2. Calcula, utilizando un ciclo *for*, la suma de los primeros $100$ números pares.
3. ¿En qué posición se encuentra el número $58$ dentro de la sucesión?

In [95]:
pares=[]
for i in range(1,101):
    pares.append(i*2)
print(pares)   
print(pares[9])

[2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22, 24, 26, 28, 30, 32, 34, 36, 38, 40, 42, 44, 46, 48, 50, 52, 54, 56, 58, 60, 62, 64, 66, 68, 70, 72, 74, 76, 78, 80, 82, 84, 86, 88, 90, 92, 94, 96, 98, 100, 102, 104, 106, 108, 110, 112, 114, 116, 118, 120, 122, 124, 126, 128, 130, 132, 134, 136, 138, 140, 142, 144, 146, 148, 150, 152, 154, 156, 158, 160, 162, 164, 166, 168, 170, 172, 174, 176, 178, 180, 182, 184, 186, 188, 190, 192, 194, 196, 198, 200]
20


1. ¿Cuál es el valor del décimo término?


El valor del decimo termino es de 18

2. Calcula, utilizando un ciclo *for*, la suma de los primeros $100$ números pares.


In [90]:
suma=0
for num in pares:
    suma+=num
print(suma)


10100


3. ¿En qué posición se encuentra el número $58$ dentro de la sucesión?

In [94]:
print(pares.index(58))

29


# Problema 5

Un nuevo software tiene $500$ usuarios y se espera que el número de usuarios crezca un $15\%$ respecto al mes anterior. Considere que $b_{n}$ es el número de usuarios transcurridos $n$ meses.

1. Calcula cuántos usuarios tendrá el software al cabo de tres meses. ¿Y al cabo de seis meses?
2. Mediante un código en *Pyhton* muestra la cantidad de usuarios que tendrá el software cada mes durante el primer año.
2. ¿Cuántos usuarios tendrá en total el software luego de un año? Utilice ciclos *for* para calcular.

# Problema 6

Un estudiante de ingeniería informática planea ahorrar para un proyecto personal. Para esto, el primer mes deposita en una cuenta $\$12.000$, el segundo mes deposita $\$14.000$ y así sucesivamente, de modo que cada mes aumenta el depósito en $\$2.000$. Considere que $a_{i}$ corresponde al depósito del estudiante en el mes $i$.

1. Implementa un código que permita mostrar en pantalla lo que depositó mensualmente durante el primer año.
2. ¿Cuánto depositó el estudiante en febrero del segundo año?
3. ¿Cuál será el total ahorrado al finalizar dos años?

# Problema 7

Por lo general, los proyectos son organizados en varias etapas que deben completarse secuencialmente.

La fase inicial de un proyecto de software tiene una duración de $3$ meses y cada fase subsecuente reduce su duración en un $10 \%$ debido a la experiencia ganada. Determine:

1. ¿Cuánto dura la quinta fase?
2. Calcula la duración total del proyecto después de $10$ fases.

# Problema 8

En una sucesión aritmética, la diferencia entre un término y el anterior es $4$ y el décimo término es $-20$. Determine:

1. El primer término de la sucesión.
2. La expresión del término de lugar $n$.
3. Utilice un código en *Python* para calcular el término que ocupa el lugar $100$.
4. Determinar la posición del número $1.680$ en la secuencia. Calcule utilizando un ciclo *while*.

# Problema 9

El tercer término de una sucesión geométrica es $5$ y el sexto término es $40$. Determine:

1. La razón geométrica de la sucesión.
2. El primer término de la sucesión.
3. La expresión del término de lugar $n$.
4. Utilice un código en *Python* para calcular el término que ocupa el lugar $8$.
4. ¿Qué lugar ocupa en esta sucesión el término $20.480$? Calcule utilizando un ciclo *while*.

# Problema 10

Una empresa de tecnología está optimizando sus servidores. Para ello mide las transacciones por segundo (TPS) de sus servidores en cada mes. Se ha obtenido que en el segundo mes, el rendimiento del servidor fue $1,2$ miles de TPS y en el quinto mes, el rendimiento fue $2,7$ miles de TPS. Si el número de TPS que hay mes a mes sigue una sucesión geométrica, determina utilizando un código en *Python*:

1. ¿Cuál es el rendimiento el noveno mes?
2. ¿En qué mes el rendimiento es de $23,4$ miles de TPS?

Utilice dos decimales para el cálculo.

# Problema 11

Una aplicación de redes sociales se estrenó el $1$ de enero con una cierta cantidad de usuarios. A partir de ahí, el número de usuarios nuevos de cada día fue siempre el mismo. El $3$ de enero tenía $500$ usuarios en total y el $10$ de enero tenía $1.501$ usuarios totales.Determine:

1. ¿Con cuántos usuarios comenzó a funcionar la aplicación?
2. ¿Cuántos usuarios tendrá la aplicación el $31$ de enero?
3. Si la aplicación tiene $5.505$ usuarios, ¿cuántos meses lleva funcionando?



# Problema 12

Una empresa de publicidad ofrece para un puesto de trabajo un salario inicial y un aumento mensual en una razón constante. Javier se presenta a la entrevista, y el encargado del departamento de recursos humanos le explica que en el segundo mes de trabajo su sueldo será de $\$402.000$, y que en el tercer mes de trabajo será de $\$404.010$. Determine:

1. El salario aproximado que Javier recibiría en el décimo mes de trabajo en la empresa.
2. ¿Cuál sería el ingreso total recibido por los sueldos de Javier durante el tercer año de trabajo?

# Problema 13

Una empresa de ciberseguridad capacita trabajadores cada trimestre. En el primer trimestre capacitaron a $30$ trabajadores, en el segundo trimestre a $60$ trabajadores, y en el tercer trimestre a $120$ trabajadores. Si en los siguientes trimestres la empresa mantiene el mismo patrón de crecimiento, determine utilizando *Python*:

1. ¿A cuántos trabajadores capacitarán en el octavo trimestre?
2. Si la empresa lleva en el mercado $3$ años, ¿a cuántos trabajadores han capacitado en total?

# Problema 14

Al llenar un tambor con agua, se registra que durante el minuto $4$ el tambor recibe $128$ litros, y durante el minuto $9$ recibe $88$ litros. Sabiendo que en cada minuto recibe una cantidad $d$ de litros de agua menos que en el minuto anterior, determine:

1. ¿En qué instante el tambor deja de recibir agua?
2. ¿Cuántos litros recibe en total el tambor?

Realice los cálculos anteriores utilizando *Python*.